[유니온 파인드] 11월 29일

jaeseo222 opened the pull request:
1043번의 오류를 도저히 못 찾겠습니다!!! 제가 놓친 부분을 짚어주실 수 있으실까요,,ㅎㅎ,, 항상 감사합니다!

Commit:
[유니온 파인드] 11월 29일

1043번의 오류를 도저히 못 찾겠습니다!!! 제가 놓친 부분을 짚어주실 수 있으실까요,,ㅎㅎ,, 항상 감사합니다!

bsa0322 commented on 11월 23일 - 유니온 파인드/1043.cpp:
```diff
@@ -0,0 +1,77 @@
+
+	for (int i = 0; i < party.size(); i++) {
+		bool go = true;
+		for (int j = 0; j < party[i].size(); j++)
+			for (int k = 0; k < truth.size(); k++)
+				if (find(party[i][j]) == find(truth[k]))//진실을 아는 사람이 그 파티에 있을 경우
+					go = false;
```
왜 틀렸는지 찾기 힘들었네요..! 사소한,,,,실수였어요,,, 지금 반복문을 party[i].size()만큼 돌려주시는데, party[i]의 사이즈를 다시 확인해볼까요!? 문제에서 주어진 데이터보다 더 검사하고 있을지도 몰라요.

더불어, party[i]들은 모두 한 집합에 속해 있을테니, 다 검사하지 않아도 괜찮아요!

bsa0322 commented on 11월 23일 - 유니온 파인드/1976.cpp:
```diff
@@ -0,0 +1,55 @@
+}
+bool check(vector<int> plan) {
+	for (int i = 1; i < plan.size(); i++)
+		if (find(plan[i - 1]) != find(plan[i]))//입력된 도시끼리 루트가 같은 지 확인
+			return false;
```
바로 전 정점과 비교해주는 것도 좋지만, 첫 시작 정점과 현재 정점을 비교해도 괜찮아요! 그러면 배열을 꼭 사용하지 않고도 해결할 수 있어요.

bsa0322 reviewed (changes requested):
p1. 좋아요~~!! 거짓말 문제만 수정 후, 저 다시 리뷰어로 호출해주세요! 수고하셨습니다!

Commit:
[유니온 파인드] 11월 30일 - Update

ㅇ,,아니ㅣ 이걸 어떻게,,, 진짜,,, 정말 감사합니다,,, 최고십니다... 사랑해요,,,,

Commit:
[최소 신장 트리] 12월 7일 - Update

샘플 코드 보며 공부하였습니다! 항상 처음에 구현 문제를 맞닥뜨리면 뭐부터 해야 할 지 막막하게만 느껴지는데, 샘플 코드에서 차근차근히 설명해주셔서 너무 감사드리고 감을 잡게 되는 것 같아요! 항상 좋은 코드와 좋은 수업, 좋은 피드백 감사드립니다!
